simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/ChangeAvailabilityRequestHandler.java

```diff
@@ -71,90 +55,16 @@ public void handle(String callId, ChangeAvailabilityRequest request) {
 
         EvseStatus requestedEvseStatus = availabilityStateMapper.mapFrom(request.getOperationalStatus());
 
-        if (changeEvseAvailability(request)) {
-
-            Optional<Evse> evse = stationState.tryFindEvse(request.getEvseId());
-
-            if (evse.isPresent()) {
-
-                sendResponseWithStatus(callId, defineStatusToSend(requestedEvseStatus, evse.get()));
-
-                sendNotificationRequest(evse.get());
-
-            } else {
-
-                sendResponseWithStatus(callId, REJECTED);
-
-            }
-
+        if (isEvseAvailability(request)) {
```

Comment:
Minor: i would replace this method call with request.getEvseId() != 0 to match the comment, especially since the private method only has one if.

Reply:
this method improves readability, expression request.getEvseId() != 0 does not give any meaningful information, javadocs meant to explain the logic

Reply:
should the method name be then isChangeEvseAvailability?

Reply:
done